kolibri/plugins/facility/assets/src/routes.js

```diff
@@ -124,4 +127,19 @@ export default [
       }
     },
   },
+  {
+    path: '/:facility_id?/:device_id?/managesync',
+    component: ManageSyncSchedule,
+    name: 'ManageSyncSchedule',
+  },
+  {
+    path: '/:facility_id?/:device_id?/editsyncschedule/?',
```

Comment:
I think we need the device_id here, so it's not an optional parameter, and I don't think we need the `?` at the end?
`path: '/:facility_id?/:device_id/editsyncschedule/',`

Reply:
Removing the` ? `on the device_id prevents the page from loading

Reply:
I added a comment here about how you'd need to update the navigation to fix this (and updates in reading the deviceId too): https://github.com/learningequality/kolibri/pull/9786/files#r1120769310